src/stdlib/map_util.bpf.c

```diff
@@ -0,0 +1,11 @@
+#define __KERNEL__
+#include <linux/types.h>
+
+#include <bpf/bpf_helpers.h>
+
+int __has_key(void * map, void * key) {
```

Comment:
Nit: this API here is a bit confusing. Maybe you want to return `_Bool` again like the other case? It returns `1` if it does *not* exist, and zero if it does. If you want to do that, I'd make it clear that it returns either zero or an errno, and return e.g. `-EEXIST` or something else rather than `1`.

Reply:
Ok, I realized I needed to add boolean support for our interop with BTF. Please let me know if it looks ok.